src/CheckoutSdk/Disputes/Dispute.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Checkout.Common;
+
+namespace Checkout.Disputes
+{
+    public class Dispute : Resource
+    {
+        /// <summary>
+        /// Gets or sets the unique identifier of the dispute.
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the reason for the dispute. <see href="https://docs.checkout.com/docs/disputes#section-dispute-reasons-and-recommended-evidence">Find out more</see>.
+        /// </summary>
+        public string Category { get; set; }
+
+        /// <summary>
+        /// Gets or sets the current status of the dispute.
+        /// </summary>
+        public string Status { get; set; }
+
+        /// <summary>
+        /// Gets or sets the amount that is being disputed, in the processing currency.
+        /// </summary>
+        public float Amount { get; set; }
```

Comment:
Should be decimal

Reply:
Ok. Why is this different from our usual amount type?

Reply:
Making it `long` as per personal catch up 😉